packages/documentation/src/stories/foundation/color/color.stories.mdx

```diff
@@ -0,0 +1,105 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import './color.styles.scss';
+import { ColorSwatch } from './color-swatch';
+
+<Meta
+  title="Foundations/Color"
+  parameters={{
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+  }}
+/>
+
+# Color
+
+You can find the [color usage documentation](#usage) right below.
+
+## Brand colors
+
+<section className="color-square__section">
+  {['yellow', 'black', 'white'].map(color => (
+    <ColorSwatch key={color} color={color} />
+  ))}
+</section>
+
+## Shades of gray
+
+<section className="color-square__section">
+  {['gray-10', 'gray-20', 'gray-40', 'gray-60', 'gray-80'].map(color => (
+    <ColorSwatch color={color} key={color} isBackground={false} />
+  ))}
+</section>
+
+## Backgrounds
+
+<section className="color-square__section">
+  {['light', 'gray'].map(color => {
+    return <ColorSwatch key={color} color={color} />;
+  })}
+</section>
+
+## Signal colors
+
+<section className="color-square__section">
+  {['success', 'danger', 'warning', 'info'].map(color => {
+    return <ColorSwatch key={color} color={color} />;
+  })}
+</section>
+
+## Dark accent colors
+
+<section className="color-square__section">
+  {['nightblue', 'petrol', 'coral', 'olive', 'purple', 'aubergine'].map(color => {
+    return <ColorSwatch key={color} color={color} />;
+  })}
+</section>
+
+## Bright accent colors
+
+<section className="color-square__section">
+  {[
+    'nightblue-bright',
+    'petrol-bright',
+    'coral-bright',
+    'olive-bright',
+    'purple-bright',
+    'aubergine-bright',
+  ].map(color => {
+    return <ColorSwatch key={color} color={color} />;
+  })}
+</section>
+
+# Usage
+
+## CSS
+
+Import the [Design System Styles package](https://www.npmjs.com/package/@swisspost/design-system-styles) and choose from one of the provided style files:
+
+- **basics.css**: Just the basic styling for resets, typography and utility classes, components must be imported separately
+- **index.css**: The complete styling for internet facing applications
+- **intranet.css**: The complete styling for internal applications
```

Comment:
We should not link to the styles package here.
This is something we could describe on a install docs page for example.

And I thinks it would be more helpful for DEVs, if we provide the import statements directly here.
I would expect something like the following in the mdx file:

<pre>
Import one of the provided style files.

To import just the basic styling for resets, typography and utility classes:
```css
import '@swisspost/design-system-styles/basics.css';
```
</pre>